README-FR.md

```diff
@@ -0,0 +1,133 @@
+# if me
```

Comment:
Each README file has a header section with the badges and links to the other translations:

<img width="598" alt="image" src="https://user-images.githubusercontent.com/3010728/66225737-011e0580-e68e-11e9-8ef8-99c85f383380.png">

You'll want to add this and also add a link to this file in the other READMEs.